indico/modules/events/tracks/models/tracks.py

```diff
@@ -0,0 +1,60 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from indico.core.db.sqlalchemy import db
+from indico.core.db.sqlalchemy.descriptions import DescriptionMixin
+from indico.util.string import format_repr, return_ascii
+
+
+class Track(DescriptionMixin, db.Model):
+    __tablename__ = 'tracks'
+    __table_args__ = {'schema': 'events'}
+
+    #: the unique id of the track
```

Comment:
i stopped it some time ago - even in autogenerated docs it's a bit pointless to document columns which are completely self-explanatory